### PR TITLE
Create project homepage and documentation using docusaurus

### LINKED ISSUE #31: Create project homepage and documentation using Docusaurus
### What would you like to be added?

I would like to integrate Docusaurus into our project to create a dedicated homepage and documentation section. Docusaurus provides a clean and user-friendly interface for managing documentation and showcasing project information.

### Why is this needed?

Having a well-organized project homepage and documentation is essential for providing users with easy access to information about the project, its features, installation guides, and API references. Docusaurus will help us maintain consistency in styling and layout across these sections.

### PR DESCRIPTION
**Pull request by ykdy3951**
#### What type of PR is this?:
documentation, enhancement

#### What this PR does / why we need it:
This PR aims to set up a project homepage and documentation using Docusaurus.

#### Which issue(s) this PR fixes:

Fixes #31 

#### Special notes for your reviewer:
Docusaurus is a popular tool for building open source project websites and documentation. It offers a clean, customizable, and easy-to-maintain structure for organizing project content. By leveraging Docusaurus, we can enhance the visibility and accessibility of our project to the community and potential users.

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation:

```docs

```

### CODE REVIEW

File: cllm-website/blog/2024-03-15-welcome/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Hello and Welcome to the **CLLM** documentation!
+
+CLLM is a **C**ommand **L**ine **L**arge Language **M**odel tool that is designed to help you write cli-command.
+Currently, CLLM is based on **GPT-3.5 Turbo** Model, which is a large language model developed by OpenAI.
+
+Below are the list of commands that you can use to interact with CLLM.
+- `help` - To get the list of commands that you can use.
+- `search` - To search for a specific command about your query.
+- `set key` - To set the OPEN API key for using CLLM.
```
Comment (devleejb) [inline cllm-website/blog/2024-03-15-welcome/index.md:16]: It should be `OpenAI API Key`.

File: cllm-website/blog/authors.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+chanism:
+  name: Chanwoong Kim
+  title: Maintainer of CLLM
```
Comment (devleejb) [inline cllm-website/blog/authors.yml:3]: It should be `Owner`.

File: cllm-website/docs/commands/set/key.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 1
+---
+
+# key
+
+The `key` command is used to set the API key for the LLM API.
```
Comment (devleejb) [inline cllm-website/docs/commands/set/key.md:7]: I think that it's better to add wording about `OpenAI API Key`.

File: cllm-website/docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+```bash
+brew tap dev-backpack/cllm
+brew install cllm
+```
+
+### Install Script
+
+```bash
+curl -sSL https://raw.githubusercontent.com/dev-backpack/cllm/main/install.sh | sh
+```
```
Comment (devleejb) [inline cllm-website/docs/getting-started.md:24]: I think that it's good to add `OS`.

File: cllm-website/docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+---
+
+# Getting Started
+
+Welcome to the **CLLM** documentation! This guide will help you get started with **CLLM** and show you how to use it to improve your CLI experience.
```
Comment (devleejb) [inline cllm-website/docs/getting-started.md:7]: How about add short description about cllm?

File: cllm-website/static/img/cllm-example-1.gif
Comment (devleejb) [inline cllm-website/static/img/cllm-example-1.gif:1]: Should we add two same gif?

File: cllm-website/README.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
+
+```
+$ yarn start
```
Comment (devleejb) [inline cllm-website/README.md:14]: I think it's better to change it to `npm`.

**Review by devleejb — CHANGES_REQUESTED** [review]

File: cllm-website/docs/getting-started.md (hunk 5 above)
Comment (ykdy3951) [reply cllm-website/docs/getting-started.md:7]: `Out Docs` has already short description about cllm in introduction tab, I think this is not necessary in `Getting Started` tab.
I want to know your opinion

**Review by devleejb — APPROVED** [review]

**Pull request merged**